pubspec.yaml

```diff
@@ -1,6 +1,6 @@
 name: store_redirect
 description: A Flutter plugin to redirect users to an app page in Google Play Store and Apple App Store.
-version: 1.0.2
+version: 2.0.0
```

Comment:
I would not bump the major version for a null check.
Any specific reason?

Reply:
Tbh I would like a major version bump because null safety is a breaking change.

Reply:
Most other packages have done a major version change, as this is a breaking change.
This would also fall in line with the [Semantic Versioning guidelines](https://semver.org/) which are most often used for versioning.

Reply:
Indeed, I agree.